src/main/java/entities/SuggestedTitleCheckerBasic.java

```diff
@@ -0,0 +1,152 @@
+package entities;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * The implementation of Suggested Title Checker for the Suggest Title use case. Checks whether a given title is
+ * a valid story title. This class has a default constructor
+ */
+public class SuggestedTitleCheckerBasic implements SuggestedTitleChecker {
+    static ArrayList<Character> VALID_PUNCTUATION =
+            new ArrayList<Character>(Arrays.asList(',','.','"','\'',':',';',')','(','/','-',' '));
+    static int MINIMUM_TITLE_LENGTH = 10;
```

Comment:
Not pressing, but why not allow for titles shorter than 10 characters? There could be one word titles that make sense

Reply:
minimum length has been reduced to 3. i think this is good since titles of length 1 or 2 are likely to not be very coherent.